lean.py

```diff
@@ -76,15 +76,16 @@ def getErrorMessage(out):
 
 def checkLean(lean_code_block):
     proofsearch = ProofSearch(path_to_repl="repl")
-    try:    
+    try:
         out = proofsearch.run_code(lean_code_block.strip(), verbose=True)
     except pexpect.exceptions.EOF:
         return {
             "status": 1,
             "num_first_line": 0,
             "error": ""
         }
-    error_message = getErrorMessage(out)
+    if out:  # failed due to timeout
```

Comment:
this is not necessary since we return from timeout?

Reply:
not currently returning on timeout in ProofSearch - there is only a print statement